direct/src/filter/FilterManager.py

```diff
@@ -307,29 +329,47 @@ def createBuffer(self, name, xsize, ysize, texgroup, depthbits=1, fbprops=None):
             props.addProperties(fbprops)
 
         depthtex, colortex, auxtex0, auxtex1 = texgroup
-        if (auxtex0 != None):
+        if auxtex0 is not None:
             props.setAuxRgba(1)
-        if (auxtex1 != None):
+        if auxtex1 is not None:
             props.setAuxRgba(2)
-        buffer=base.graphicsEngine.makeOutput(
+        buffer = base.graphicsEngine.makeOutput(
             self.win.getPipe(), name, -1,
             props, winprops, GraphicsPipe.BFRefuseWindow | GraphicsPipe.BFResizeable,
             self.win.getGsg(), self.win)
-        if (buffer == None):
+        if buffer is None:
             return buffer
-        if (depthtex):
-            buffer.addRenderTexture(depthtex, GraphicsOutput.RTMBindOrCopy, GraphicsOutput.RTPDepth)
-        if (colortex):
-            buffer.addRenderTexture(colortex, GraphicsOutput.RTMBindOrCopy, GraphicsOutput.RTPColor)
-        if (auxtex0):
-            buffer.addRenderTexture(auxtex0, GraphicsOutput.RTMBindOrCopy, GraphicsOutput.RTPAuxRgba0)
-        if (auxtex1):
-            buffer.addRenderTexture(auxtex1, GraphicsOutput.RTMBindOrCopy, GraphicsOutput.RTPAuxRgba1)
+        if depthtex:
+            buffer.addRenderTexture(
+                depthtex,
+                GraphicsOutput.RTMBindOrCopy,
+                GraphicsOutput.RTPDepth
+            )
+        if colortex:
+            buffer.addRenderTexture(
+                colortex,
+                GraphicsOutput.RTMBindOrCopy,
+                GraphicsOutput.RTPColor
+            )
+        if auxtex0:
+            buffer.addRenderTexture(
+                auxtex0,
+                GraphicsOutput.RTMBindOrCopy,
+                GraphicsOutput.RTPAuxRgba0
+            )
+        if auxtex1:
+            buffer.addRenderTexture(
+                auxtex1,
+                GraphicsOutput.RTMBindOrCopy,
+                GraphicsOutput.RTPAuxRgba1
+            )
         buffer.setSort(self.nextsort)
         buffer.disableClears()
         self.nextsort += 1
         return buffer
 
+    # Disable unused variable warning to avoid API change
+    # pylint: disable=unused-argument
     def windowEvent(self, win):
```

Comment:
You can change `win` to `_win` to mark the variable as intentionally unused without breaking the API. The method will still take two arguments, and since no keyword arguments are used, renaming the argument should not be a problem either.

Reply:
Thanks, forgot about that trick. Fixed.